<?xml version="1.0"?>
<launch>
    <node pkg="waypoint_updater" type="waypoint_updater.py" name="waypoint_updater" />
    <param name="velocity" value="10" />
    <param name="limit_dist" value="100" />
    <param name="brake_dist" value="60" />
    <param name="hard_limit_dist" value="8" />
</launch>
